rest/utilities_testing.go

```diff
@@ -231,7 +230,39 @@ func (rt *RestTester) Bucket() base.Bucket {
 
 		rt.DatabaseConfig.SGReplicateEnabled = base.BoolPtr(rt.RestTesterConfig.sgReplicateEnabled)
 
-		_, err = rt.RestTesterServerContext.AddDatabaseFromConfig(*rt.DatabaseConfig)
+		if rt.RestTesterConfig.TestBucket != nil {
+			// If using a leaky bucket, make sure it ignores attempted closures to avoid double closures panic
+			// due to closing both RestTesterServerContext and testBucket in rt.Close()
+			leakyBucket, isLeaky := base.AsLeakyBucket(testBucket)
+			if isLeaky {
+				leakyBucket.SetIgnoreClose(true)
+			}
+
+			// Set scopes and collections on bucket if set in DB Config
```

Comment:
The comment from `MakeBucketSpec` is clearer about this being a temporary thing

```suggestion
			// WIP: Collections Phase 1 - Grab just one scope/collection from the defined set.
			// Phase 2 (multi collection) means DatabaseContext needs a set of BucketSpec/Collections, not just one...
```